app/views/admin/organizations/_loans.html.slim

```diff
@@ -1,4 +1,11 @@
-h1 = t('organization.active_loans')
+/ JE Todo: Confirm how these styles should be factored
+div style="margin-bottom: 15px"
+  h1 style="display: inline"
+    = t('organization.active_loans')
+  span style="margin-left: 10px"
+    = link_to t(".view_all_loans"), admin_loans_path({'loans[f][organizations.name][]' => @org.name})
+  span style="float: right"
+    = link_to t("loan.shared.new"), new_admin_loan_path({organization_id: @org.id}), class: 'btn btn-default new-button'
```

Comment:
Going to get input from Liz in a min on how to handle styles. Please stay tuned.